<launch>
    <arg name="model_name" default="TunnelYawPredictor-_bs128_ne128_lr0_000875.torch" />
    <arg name="model_folder" default="$(env HOME)/models/yaw_estimation" />
    <arg name="path_to_model" default="$(arg model_folder)/$(arg model_name)" />
    <arg name="input_topic" default="" />
    <arg name="output_topic" default="" />
    <arg name="model_module" default="" />
    <arg name="model_type" default="" />
    <arg name="publish_also_in_deg" default="" />
    <node pkg="tunnel_yaw_prediction_ros" type="tunnel_yaw_prediction_node.py"
        name="tunnel_yaw_prediction_node" output="screen"
        args="--path_to_model $(arg path_to_model) --input_topic $(arg input_topic) --output_topic $(arg output_topic) --model_module $(arg model_module) --model_type $(arg model_type) --publish_also_in_deg $(arg publish_also_in_deg) --rosargs" />
</launch>